modules/API/API.js

```diff
@@ -89,6 +89,11 @@ function initCommands() {
 
             APP.store.dispatch(muteAllParticipants(localIds));
         },
+        'enable-lobby': () => {
```

Comment:
How about disabling it? This needs to be a "toggle" command so the user can decide if they want to enable or disable it.

Reply:
Done. Changed it to a toggle instead of just enabling the mode.